Nodejs/Product/Nodejs/Commands/ImportWizardCommand.cs

```diff
@@ -81,7 +81,15 @@ class ImportWizardCommand : Command {
                         }
                         if (File.Exists(path)) {
                             object outRef = null, pathRef = "\"" + path + "\"";
-                            NodejsPackage.Instance.DTE.Commands.Raise(VSConstants.GUID_VSStandardCommandSet97.ToString("B"), (int)VSConstants.VSStd97CmdID.OpenProject, ref pathRef, ref outRef);
+                            var addExistingProjectCmd = NodejsPackage.Instance.DTE.Commands.Item("File.AddExistingProject");
```

Comment:
Does this work in the instance where a project is open (so the AddExistingProject command is available), but someone is nonetheless creating a new project?

Perhaps a better solution is to intercept the command in the NodejsPackage.Initialize() step by handling BeforeExecute and AfterExecute events (set a boolean flag that we check in ImportWizardCommand). 
E.g. http://stackoverflow.com/questions/15406523/visual-studio-extensibility-intercepting-file-paste-event